### PR TITLE
ADEN-4086 Change VastUrlBuilder interface

### PR DESCRIPTION
**Pull request by fraszczakszymon**
So it accepts all given slot params

### CODE REVIEW

File: extensions/wikia/AdEngine/js/video/vastUrlBuilder.js [hunk 1]
```diff
@@ -68,3 +72,12 @@ define('ext.wikia.adEngine.video.vastBuilder', [
 		build: build
 	};
 });
+
+// TODO: ADEN-4128 - remove me
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/video/vastUrlBuilder.js:76]: INFO Complete the task associated to this TODO comment. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
- INFO 1 info

Watch the comments in this conversation to review them.

File: extensions/wikia/AdEngine/js/video/vastUrlBuilder.js [hunk 2]
```diff
@@ -40,18 +42,20 @@ define('ext.wikia.adEngine.video.vastBuilder', [
...
+		slotParams = slotParams || {};
+
 		var params = [
 				'output=vast',
 				'env=vp',
 				'gdfp_req=1',
 				'impl=s',
 				'unviewed_position_start=1',
-				'iu=' + (src && slotName ?  adUnitBuilder.build(slotName, src) : defaultAdUnit),
+				'iu=' + adUnitBuilder.build(slotParams.pos, slotParams.src),
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/video/vastUrlBuilder.js:54]: no need to fallback to something if pos or src is not set?

File: extensions/wikia/AdEngine/js/video/vastUrlBuilder.js [hunk 3]
```diff
@@ -40,18 +42,20 @@ define('ext.wikia.adEngine.video.vastBuilder', [
...
 				'sz=' + getSize(aspectRatio),
 				'url=' + loc.href,
 				'correlator=' + correlator,
-				'cust_params=' + getCustomParameters(slotName, src)
+				'cust_params=' + getCustomParameters(slotParams)
```
Comment (bognix) [inline extensions/wikia/AdEngine/js/video/vastUrlBuilder.js:58]: 👍

**Comment by bognix** [comment]
unit tests failure not related to the code change

**Review by bognix — APPROVED** [review]
LGTM

File: extensions/wikia/AdEngine/js/video/vastUrlBuilder.js (hunk 2 above)
Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/video/vastUrlBuilder.js:54]: I believe it shouldn't happen. Anyway fallback to `VIDEO_ATG` was meaningless

**Comment by bognix** [comment]
Travis failure related with github issues

**Pull request merged**